test/show.jl

```diff
@@ -35,4 +38,7 @@ module TestShow
     show(io, A)
     A = DataFrames.RepeatedVector([1, 2, 3], 1, 5)
     show(io, A)
+
+    render(Juno.Editor(), df)
```

Comment:
Ok I've added a test for the implementation. Given that there's no logic it's equivalent to checking the source code hasn't changed, but whatever, it's a test.